frontend/src/components/dashboard/DescLinks.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Box from '@material-ui/core/Box';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import Typography from '@material-ui/core/Typography';
+import LinkIcon from '@material-ui/icons/Link';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: '100%',
+    maxWidth: 500,
+  },
+}));
+
+function ListItemLink(props) {
+  return <ListItem button component="a" {...props} />;
+}
+
+export default function DescLinks() {
+  const classes = useStyles();
+
+  return (
+    <>
+      <Box m={4} />
```

Comment:
A breakpoint for medium screen is "md".  I ran across the same thing. It just silently fails.